app/components/orders/company-info.js

```diff
@@ -0,0 +1,3 @@
+import Component from '@ember/component';
+
+export default Component.extend({});
```

Comment:
Empty component JS is not needed, delete

Reply:
done

Reply:
Sir, please test deployment one time,  because everything is Working well in my local machine,  but an error comes on deployment.